index.js

```diff
@@ -6,15 +6,19 @@ const utils = require('./lib/utils');
 const processors = require('./lib/processors');
 const listEndpoints = require('express-list-endpoints');
 
-const packageJsonPath = `${process.cwd()}/package.json`;
-const packageInfo = fs.existsSync(packageJsonPath) ? require(packageJsonPath) : {};
+let packageJsonPath = `${process.cwd()}/package.json`;
+let packageInfo;
```

Comment:
This change breaks module `packageInfo` by deafult won't be initialized and below `packageInfo.baseUrlPath` will be always undefined. `packageInfo` won't be initialized at least because nobody from dedelopers know about `updateSpecFromPackage` - it is not described in the README.md. But rather then makes live of developers harder I would call `updateSpecFromPackage` here so we know for sure that `packageInfo` inntialized with default values.